.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,9 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <codeStyleSettings language="JAVA">
+      <indentOptions>
+        <option name="USE_TAB_CHARACTER" value="true" />
+      </indentOptions>
+    </codeStyleSettings>
+  </code_scheme>
+</component>
```

Comment:
IDE-specific file. I suppose this isn't in the gitignore, nor is much else for IntelliJ. Remove this file and I'll update the .gitignore to be much more inclusive of IDE files after this is merged